### ISSUE TITLE
I made a copy of parts of the server before it shutdown

### ISSUE THREAD

**Issue by RobustMarker** [Milxnor/Project-Reboot-3.0#98]
I will upload the files to a repo and link them.
The following channels were fully backed up (including media): announcements, info, launcher and updates.

EDIT:
The copy of all files are up in my repo called ProjectRebootDiscordBackup
https://github.com/RobustMarker/ProjectRebootDiscordBackup

**Comment by Milxnor** [comment]
This is actually really awesome I wish someone coulda got general but that would have been massive

**Comment by NazgulCoder** [comment]
> This is actually really awesome I wish someone coulda got general but that would have been massive

Actually in order to avoid future problems you should really consider selfhosting Revolt (which is a Discord clone)

**Comment by RobustMarker** [comment]
> Actually in order to avoid future problems you should really consider selfhosting Revolt (which is a Discord clone)

Yes and no, revolt is better in discord in some ways as the bar is set so low by discord in the first place but revolt is plagued with issues and its owner having some sketchy views. Revolt isnt even fully open source.

Unfortunately in this modern day and age we cant take anything for granted and will have to just archive stuff from whatever platform so we have a guaranteed way to reivew it later. 

Im not aware of any other discord alts apart from revolt and guilded.

**Comment by NazgulCoder** [comment]
> > Actually in order to avoid future problems you should really consider selfhosting Revolt (which is a Discord clone)
> 
> Yes and no, revolt is better in discord in some ways as the bar is set so low by discord in the first place but revolt is plagued with issues and its owner having some sketchy views. Revolt isnt even fully open source.
> 
> Unfortunately in this modern day and age we cant take anything for granted and will have to just archive stuff from whatever platform so we have a guaranteed way to reivew it later.
> 
> Im not aware of any other discord alts apart from revolt and guilded.

What about Spacebar or Matrix?

Spacebar seems very similar to Discord/Revolt, FOSS and promising. For something more stable maybe Matrix?

Anyway I don't wanna disturb anymore, keep up with the good work :)

**Comment by Milxnor** [comment]
Its hard enough to get members on discord unfortunately, any other platform would be even more of a struggle
